### ISSUE TITLE
Lazy not detected as plugin manager

### ISSUE THREAD

**Issue by Maritsu** [andweeb/presence.nvim#145]
### Description

Opening the Lazy interface does not change the RPC text to the plugin manager state

### Neovim version

```markdown
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1702233742
```

### OS information

EndeavourOS x86_64

### Steps to reproduce

1. Set up `presence.nvim` using Lazy:
```lua
  -- Insert this into Lazy plugins list
  {
    'andweeb/presence.nvim',
    event = "VeryLazy",
    opts = {},
  },
```
2. Open Lazy via `:Lazy` or any keybind mapped to that
3. Plugin manager text does not show up

### Logs

```
[presence.nvim] Setting up plugin for linux
[presence.nvim] Using user-defined Discord client id
[presence.nvim] Using runtime path: /run/user/1000
[presence.nvim] Using Discord IPC socket path: /run/user/1000/discord-ipc-0
[presence.nvim] Checking Discord IPC socket at /run/user/1000/discord-ipc-0...
[presence.nvim] Using id [redacted]
[presence.nvim] Completed plugin setup
[presence.nvim] Getting nvim socket paths...
[presence.nvim] Executing command: `ss -lx|grep "nvim.*/0"`
[presence.nvim] Checked Discord IPC socket, looks good!
[presence.nvim] Checked Discord IPC socket, looks good!
[presence.nvim] Handling TextChanged event...
[presence.nvim] Connecting to Discord...
[presence.nvim] Connected to Discord
[presence.nvim] Authorizing with Discord...
[presence.nvim] Wrote message to pipe
[presence.nvim] Got nvim socket paths: {}
[presence.nvim] No other remote nvim instances
[presence.nvim] Got nvim socket paths: {}
[presence.nvim] No other remote nvim instances
[presence.nvim] Handling WinLeave event...
[presence.nvim] Handling WinEnter event...
[presence.nvim] Handling BufEnter event...
[presence.nvim] Currently authorizing with Discord, scheduling callback for later...
[presence.nvim] Not canceling presence due to switching to a window within the same instance...
[presence.nvim] Currently authorizing with Discord, scheduling callback for later...
[presence.nvim] Authorized with Discord for [redacted]
[presence.nvim] Parsed filename nil with no extension
[presence.nvim] Getting project name for nil...
[presence.nvim] Setting activity for unnamed buffer...
[presence.nvim] No project detected
[presence.nvim] Sync activity to all peers...
[presence.nvim] Setting Discord activity...
[presence.nvim] Activity already set for , skipping...
[presence.nvim] Activity already set for , skipping...
[presence.nvim] Wrote message to pipe
[presence.nvim] Handling TextChanged event...
[presence.nvim] Last activity sent was within 10 seconds ago, skipping...
[presence.nvim] Set activity in Discord for nil
[presence.nvim] Handling WinLeave event...
[presence.nvim] Handling WinEnter event...
[presence.nvim] Handling BufEnter event...
[presence.nvim] Not canceling presence due to switching to a window within the same instance...
[presence.nvim] Activity already set for , skipping...
[presence.nvim] Activity already set for , skipping...
[presence.nvim] Handling BufAdd event...
[presence.nvim] Handling BufEnter event...
[presence.nvim] Handling BufAdd event...
[presence.nvim] Handling BufAdd event...
[presence.nvim] Handling BufAdd event...
[presence.nvim] Handling BufAdd event...
[presence.nvim] Handling BufEnter event...
[presence.nvim] Parsed filename plugins.lua with lua extension
[presence.nvim] Getting project name for /home/bit/src/wm/nvim/lua/custom...
[presence.nvim] Setting activity for /home/bit/src/wm/nvim/lua/custom/plugins.lua...
[presence.nvim] Using file asset: { "Lua", "lua" }
[presence.nvim] Detected project: nvim
[presence.nvim] Sync activity to all peers...
[presence.nvim] Setting Discord activity...
[presence.nvim] Activity already set for /home/bit/src/wm/nvim/lua/custom/plugins.lua, skipping...
[presence.nvim] Activity already set for /home/bit/src/wm/nvim/lua/custom/plugins.lua, skipping...
[presence.nvim] Wrote message to pipe
[presence.nvim] Activity already set for /home/bit/src/wm/nvim/lua/custom/plugins.lua, skipping...
[presence.nvim] Activity already set for /home/bit/src/wm/nvim/lua/custom/plugins.lua, skipping...
[presence.nvim] Activity already set for /home/bit/src/wm/nvim/lua/custom/plugins.lua, skipping...
[presence.nvim] Activity already set for /home/bit/src/wm/nvim/lua/custom/plugins.lua, skipping...
```

**Comment by FelipeFTN** [comment]
@Maritsu Are you still having problems?
I did just like this, and Lazy recognized just fine:
```
  -- Discord presence
  {
    "andweeb/presence.nvim",
    opts = overrides.presence,
    lazy = false,
  },
```
Try setting `lazy = false` for the extension to load properly right after start NeoVim.

**Comment by Maritsu** [comment]
> @/Maritsu Are you still having problems?
> I did just like this, and Lazy recognized just fine:
> ```
>   -- Discord presence
>   {
>     "andweeb/presence.nvim",
>     opts = overrides.presence,
>     lazy = false,
>   },
> ```
> Try setting `lazy = false` for the extension to load properly right after start NeoVim.

I had previoisly set it to `lazy = true` with `event = "UIEnter"` and simply submitted a PR adding Lazy to some table, but I'll try this. Thanks!

**Comment by jiriks74** [comment]
Hello,
if you'd like you can transfer this issue to [my repository](https://github.com/jiriks74/presence.nvim) and we can take a look at it.

This repository seems to be completely dead so I'm trying to fix some things and merge contributions. If you'd like to see a rough plan of what I want to do you can look at my [OpenProject instance](https://openproject.stefka.eu/projects/3/?jump=home).

**Comment by Maritsu** [comment]
closed by jiriks74#39
